pkgs/development/tools/revive/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "revive";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mgechev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-eQE0kkiDynVoPqldQD3cHCxC0Q94gSdkbMFy9y6Rqp4=";
+    # populate values that require us to use git. By doing this in postFetch we
+    # can delete .git afterwards and maintain better reproducibility of the src.
+    leaveDotGit = true;
+    postFetch = ''
+      date -u '+%Y-%m-%d %H:%M UTC' > $out/DATE
```

Comment:
this is going to cause issues. you need to generate the date via the commit date since that wont change

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/tools/admin/syft/default.nix#L18-L19

Reply:
👍 Fixed